common-design-patterns/global-header.md

```diff
@@ -0,0 +1,426 @@
+          </ul>
+        </details>
+        <details id="005">
+          <summary><strong>CDTS</strong></summary>
+          <p class="mrgn-tp-lg">For the Centrally Deployed Templates Solution (CDTS):</p>
+          <ul>
+            <li><a href="https://cenw-wscoe.github.io/sgdc-cdts/docs/index-en.html">CDTS documentation</a></li>
+          </ul>
+        </details>
+        <details id="006">
+          <summary><strong>Drupal WxT</strong></summary>
+          <p class="mrgn-tp-lg">For Drupal WxT:</p>
+          <ul>
+            <li><a href="https://drupalwxt.github.io/en/">Drupal WxT documentation</a></li>
+          </ul>
+        </details>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="cnt-wdth-lmtd">
+  <h2 id="research">Research and rationale</h2>
+  <p>Consult research findings and policy rationale.</p>
+  <h3>Research findings</h3>
+  <p>Two research projects informed the latest updates to the header guidance.</p>
+  <h4>Trust Signals for Canada.ca study</h4>
+  <p>Our Canada.ca trust study and prior research show that a consistent header is necessary to maintaining a trusted brand.</p>
+  <p>For example, people trust:</p>
+  <ul>
+    <li>the flag in the Government of Canada signature more when it’s red</li>
+    <li>a white background more than dark mode</li>
+  </ul>
+  <p>If you want to know more about this research, contact the Digital Transformation Office at <a href="mailto:dto.btn@tbs-sct.gc.ca">dto.btn@tbs-sct.gc.ca</a>.</p>
+  <h4>Wayfinding on Canada.ca project</h4>
+  <p>We updated the global header for Canada.ca to align with a new overall navigation strategy that came out of the Wayfinding research project.</p>
+  <ul>
+    <li><a href="https://blog.canada.ca/research-summaries/wayfinding-on-canada-ca">Wayfinding on Canada.ca research summary</a><br>
+      Explains the context of the research and the insights that drove the design updates.</li>
+    <li><a href="https://blog.canada.ca/2022/12/21/wayfinding-research-project">Wayfinding research project improves our approach to navigation on Canada.ca</a><br>
+      This blog post explains the changes that are being made to the Canada.ca design, and how they are being implemented.</li>
+  </ul>
+  <h3>Policy rationale</h3>
+  <p>This pattern is a mandatory element of the Content and Information Architecture Specification.</p>
+  <ul>
+    <li><a href="https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-information-architecture-specification/mandatory-elements.html">Mandatory elements of the design system</a></li>
+  </ul>
+  <h2 id="changes">Latest changes</h2>
+  <dl class="dl-horizontal">
+    <dt>
+      <time datetime="YYYY-MM-DD" class="link-muted">YYYY-MM-DD</time>
+    </dt>
+    <dd>Updated pattern to reflect design changes from the trust study and wayfinding project, added visual examples for the different types of pages, included links to research projects that inform header guidance</dd>
```

Comment:
```suggestion
    <dd>Updated pattern to reflect design changes from the trust study and wayfinding project, added advice on what to avoid, added visual examples for the different types of pages, included links to research projects that inform header guidance</dd>
```

Reply:
Done